src/cli/commands/create/app.ts

```diff
@@ -107,7 +107,7 @@ export async function runBootstrapper(
     projectRoot: fs.path(),
     sourceRoot: fs.path('./src'),
     sourceRootRelative: './src',
-    schemaModules: ['src/' + Layout.schema.FILE_NAME],
+    schemaModules: ['src/' + Layout.schema.CONVENTIONAL_SCHEMA_FILE_NAME],
```

Comment:
Identifier name change makes it easier to auto important from anywhere in the project. I am coming around to this way of naming things...